core/alert.py

```diff
@@ -140,13 +140,13 @@ def info(content, log_in_file=None, mode=None,
                 )
             )
             sys.stdout.flush()
-    # TODO: implement log functionality later
-    # if event:  # if an event is present log it
-    #     from core.log import __log_into_file
-    #     __log_into_file(log_in_file, mode, json.dumps(event), language)
-    #     # if thread temporary filename present, rewrite it
-    #     if thread_tmp_filename:
-    #         __log_into_file(thread_tmp_filename, "w", "0", language)
+
+    if event:  # if an event is present log it
+        from core.log import __log_into_file
```

Comment:
Since now we have a core/log.py file, can we not import it on the top? Or there is again a circular dependency issue?